### ISSUE TITLE
WCT displays information of the system where WinDbg is running

### ISSUE THREAD

**Issue by EveryOtherUsernameWasAlreadyTaken** [goldshtn/windbg-extensions#2]
WCT displays information of the system where WinDbg is running. It does not display information about the debugging target. E.g. when analyzing a dump, it does not display the WCT information of the dump. This was not clear to me when I downloaded the extension. If this is the expected behavior, it should just be documented.

**Comment by goldshtn** [comment]
WCT is a live API, there is no way to apply it to a dump because the required information is stored in kernel memory. Sorry if it's misleading, but that's just what the Windows API provides. 

By the way, you can reconstruct some wait information from a dump manually using the !cs and !handle commands. Doesn't have the convenience of automatic deadlock detection but definitely doable.

**Comment by EveryOtherUsernameWasAlreadyTaken** [comment]
I agree. It would be nice if there is a message during a post mortem session (e.g. analyzing a kernel dump) that this plugin only works in live debugging. Assuming you have access to the DbgEng methods, [IDebugControl::GetDebuggeeType()](https://msdn.microsoft.com/en-us/library/windows/hardware/ff546559%28v=vs.85%29.aspx) might help.
